jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -227,7 +227,8 @@ ecma_op_object_has_property (ecma_object_t *object_p, /**< the object */
  *         Returned value must be freed with ecma_free_value
  */
 ecma_value_t
-ecma_op_object_find_own (ecma_object_t *object_p, /**< the object */
+ecma_op_object_find_own (ecma_object_t *base_object_p, /**< base object */
```

Comment:
Hm, it would be good if base would be:

evma_value_t base_value;

And the reason is this:

https://github.com/Samsung/jerryscript/blob/master/jerry-core/ecma/operations/ecma-get-put-value.c#L107

I really want to rework the code there and this would provide the opportunity.